README.md

```diff
@@ -1,6 +1,193 @@
-# [WIP] metaschema-xslt
+# metaschema-xslt
 
-> NOTICE: The XSLT implementation's migration to its new home **is not complete**.
-> For more details, see [this issue](https://github.com/usnistgov/metaschema/issues/375) and [accompanying milestone](https://github.com/usnistgov/metaschema-xslt/milestone/1).
+## punchlist
+
+- [x] `CODEMETA`
+- [x] `CONTRIBUTING.md`
+- [x] `CODEOWNERS`
+- [x] `LICENSE.md`
+- [x] `CODE_OF_CONDUCT.md`
+- [x] this file (see outline/requirements at end)
+
+## License
+
+See the [project license](LICENSE.md) in this repository.
+
+This project is placed into the worldwide public domain.
+
+## Software description
+
+Metaschema-XSLT is an XSLT-based implementation of the NIST (ITL/CSD) [Metaschema Information Modeling Framework](https://github.com/usnistgov/metaschema).
```

Comment:
Metaschema Information Modeling Framework name does not appear in the metaschema repo. Should it be added in the Readme.md of metaschema for consistency?

Reply:
Isn't it in the specification?

https://pages.nist.gov/metaschema/specification/#metaschema-specification

Reply:
Yup, and all over that Pages site. Alas, maintaining that site's readme is out of scope for this Issue.